include/evmc/evmc.hpp

```diff
@@ -289,7 +289,8 @@ namespace literals
 template <typename T>
 constexpr T parse(std::string_view s) noexcept
 {
-    return from_hex<T>(s).value();
+    auto const v = from_hex<T>(s);
```

Comment:
The failure case is also used for parsing errors, including odd nibbles.

Need to rethink.